### DISCUSSION TITLE
Database Import

### DISCUSSION THREAD

**Discussion by rahul-navigate** [markshust/docker-magento#1223] [General]
How can I import "sql.gz" file using the "bin/mysql" command?

**Comment by YevhenZvieriev** [comment]
Hi, @rahul-navigate 

If you simply extract the archive and run the command, you can use the `bin/mysql` script to import the `.sql` file directly. Here’s how you can do it:

1. Extract the archive

2. Import the extracted SQL file:

`bin/mysql < ../backup.sql`

Use this command if your SQL file is within the project directory.
